pkg/control/service_control.go

```diff
@@ -113,7 +113,6 @@ func (r RealServiceControl) createServices(namespace string, service *v1.Service
 		return nil
 	}
 	log.Infof("Controller %v created service %v", accessor.GetName(), newService.Name)
-	r.Recorder.Eventf(object, v1.EventTypeNormal, SuccessfulCreateServiceReason, "Created service: %v", newService.Name)
```

Comment:
How about using flags to control whether this event should be generated.

Reply:
Done